eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -36,5 +36,5 @@ steps:
         @subscriptionConfiguration `
         -CI `
         -Force `
-        -Verbose
+        -Verbose | Out-Null
```

Comment:
Is this a change you plan to keep?

Reply:
Thanks! This also belongs in eng/common because it will output the return values of `New-TestResources.ps1` into the logs.

Reply:
We should make this PR in the tools repo so perhaps undo it here and either submit a PR or file an issue to add it there.

Reply:
Propagated in: https://github.com/Azure/azure-sdk-tools/pull/1029